src/main/webapp/voting.js

```diff
@@ -1,19 +1,26 @@
 function getInput() {
   var url_string = window.location.href;
   var url = new URL(url_string);
-  var c = url.searchParams.get("id");
-  console.log(c);
-
-  fetch("/data?id=" + id)
+  var id = url.searchParams.get("id");
+  console.log(id);
+  var route = "/id?id=" + id;
+  fetch(route)
     .then((response) => response.json())
     .then((stats) => {
-      document.getElementById("feed").innerHTML = stats.title;
-      var from = document.getElementById("vote-form");
+      console.log(stats);
+      document.getElementById("title").innerHTML = stats.question;
+      var y = document.createElement("INPUT");
+      y.setAttribute("type", "radio");
+      y.setAttribute("value", stats.option);
+      var label = document.createElement("label");
+      label.appendChild(y);
+      label.innerHTML += "<span> " + stats.option + "</span><br>";
+      document.getElementById("vote-form").appendChild(label);
 
-      for (i = 0; i < stats.options.length; i++) {
-        var y = document.createElement("INPUT");
-        y.setAttribute("type", "radio");
-        y.setAttribute("value", stats.option);
-      }
+      // for (i = 0; i < stats.option.length; i++) {
```

Comment:
Why are we commenting this code? I think we should work under the assumption that we have multiple options here and above?

Reply:
I have this code commented out so that once multiple options work then its an easy switch